packages/core/src/components/search/Filter/FilterDesktop.tsx

```diff
@@ -235,11 +201,11 @@ export const fragment = gql(`
   }
 `)
 
-const RADIO_FACETS = ['shipping', 'pickupPoint'] as const
-function isRadioFacets(str: unknown): str is (typeof RADIO_FACETS)[number] {
-  if (typeof str !== 'string') return false
+// const RADIO_FACETS = ['shipping', 'pickupPoint'] as const
```

Comment:
Do you think it's okay to delete this comment?
Did we moved it to the `sdk/useDeliveryPromise`?

Reply:
Yes, just forgot to remove 😬